app/src/main/java/com/example/fitlife/PersonProfileActivity.java

```diff
@@ -33,12 +35,12 @@ public class PersonProfileActivity extends AppCompatActivity {
 
     TextView first, last, email, user;
     CircleImageView profileImage;
-    Button sendFriendRequest, declineFriendRequest;
+    Button sendFriendRequest, declineFriendRequest, group;
     FirebaseAuth fAuth;
     DatabaseReference reference, userRef, friendsRef;
     FirebaseDatabase fData;
     StorageReference storageReference;
-    String sendUserId, recieverUserId, currentState, saveCurrentDate;
+    String sendUserId, recieverUserId, currentState, saveCurrentDate, challenge;
 
```

Comment:
Repeated line. Have to get rid of one.

Reply:
keep String challenge, line 41 redundant